fastlane/resources/values/strings.xml

```diff
@@ -4097,7 +4097,7 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
     <string name="blogging_prompts_answer_prompt_notification_answer_action">Answer</string>
     <string name="blogging_prompts_notification_dismiss">Dismiss</string>
 
-    <!-- WordPressEditor module -->
+    <!-- Editor module -->
```

Comment:
👍  (although this would have been auto-updated during next code freeze when our automation copies the `WordPress/src/main/res/values/strings.xml` file to this `fastlane/resources/values/strings.xml` to "freeze" it… but doesn't hurt to manually update it here anyway)